analyzer/src/funTest/assets/projects/synthetic/stack-yesodweb-simple/.dir-locals.el

```diff
@@ -0,0 +1,4 @@
+((haskell-mode . ((haskell-indent-spaces . 4)
```

Comment:
My original plan was to replace (bit by bit) "real" projects that we incorporate via Git submodules with synthetic projects that are fully under our control. With that in mind, synthetic projects should actually not necessarily be as small as possible, but sufficiently large to cover the wanted use-cases. Also, to me it was attractive to use a project that can be auto-generated, which might simplify any future updates when the package manager changes its implementation.

Reply:
> With that in mind, synthetic projects should actually not necessarily be as small as possible, but sufficiently large to cover the wanted use-cases.

That's actually what I meant. However, this test project seems to be a lot larger than required.

> Also, to me it was attractive to use a project that can be auto-generated, which might simplify any future updates when the package manager changes its implementation.

That's a good argument for me to have additional files which are not required for the test. I would like if for such cases we add a `README.md` to the project directory that explains how the project was generated and how it can be updated.

Reply:
> That's a good argument for me to have additional files which are not required for the test. I would like if for such cases we add a `README.md` to the project directory that explains how the project was generated and how it can be updated.

There already is a `README.md` being added as part of the project generation, so there would be a name clash. Isn't the explanation in the Git commit message about how the projected was generated good enough?

Reply:
Yes, I think we won't be doing many manual changes to the test project so it should be easy enough to find.